assets/lang/id_ID.ini

```diff
@@ -931,7 +931,7 @@ Stopping.. = Menghentikan...
 [Reporting]
 Bad = Buruk
 FeedbackCRCCalculating = Disk CRC: Perhitungan...
-FeedbackCRCValue = Disk CRC: [VALUE]
+FeedbackCRCValue = Disk CRC: [NILAI]
```

Comment:
Don't translate `[VALUE]` or it won't work.

-[Unknown]

Reply:
Thanks for the correction, I fixed it